plugins/Microsoft.DocAsCode.Build.UniversalReference/ModelConverter/ApiBuildOutputProfile.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.UniversalReference
+{
+    using System.Collections.Generic;
+
+    using Microsoft.DocAsCode.DataContracts.Common;
+
+    using AutoMapper;
+
+    public class ApiBuildOutputProfile : Profile
+    {
+        public ApiBuildOutputProfile(string[] supportedLanguages, IReadOnlyDictionary<string, ApiNames> references = null, IReadOnlyDictionary<string, ApiBuildOutput> children = null)
+        {
+            CreateMap<ItemViewModel, ApiBuildOutput>()
+                .ForMember(dest => dest.Children, opt => opt.Ignore())
+                .ForMember(dest => dest.Parent, opt => opt.Ignore())
+                .ForMember(dest => dest.NamespaceName, opt => opt.Ignore())
+                .ForMember(dest => dest.Overridden, opt => opt.Ignore())
+                .ForMember(dest => dest.Name, opt => opt.Ignore())
+                .ForMember(dest => dest.NameWithType, opt => opt.Ignore())
+                .ForMember(dest => dest.FullName, opt => opt.Ignore())
+                .ForMember(dest => dest.Source, opt => opt.Ignore())
+                .ForMember(dest => dest.AssemblyNameList, opt => opt.Ignore())
+                .ForMember(dest => dest.Platform, opt => opt.Ignore())
+                .ForMember(dest => dest.DerivedClasses, opt => opt.Ignore())
+                .ForMember(dest => dest.Implements, opt => opt.Ignore())
+                .ForMember(dest => dest.InheritedMembers, opt => opt.Ignore())
+                .ForMember(dest => dest.ExtensionMethods, opt => opt.Ignore())
+                .ForMember(dest => dest.DerivedClasses, opt => opt.Ignore())
```

Comment:
remove line 27. line 50 is needed to convert derived classs.

---
In reply to: [114262767](https://github.com/dotnet/docfx/pull/1618#discussion_r114262767) [](ancestors = 114262767)